frontend/public/style/_common.scss

```diff
@@ -566,3 +566,21 @@ dl.co-inline {
 .no-margin {
   margin: 0 !important;
 }
+
+.co-overview-card--gradient {
+  position: relative;
+  &:after {
+    background: linear-gradient(rgba(255, 255, 255, 0), var(--pf-global--BackgroundColor--100));
+    bottom: 0;
+    content: '';
+    left: 0;
+    min-height: 2rem;
+    pointer-events: none;
+    position: absolute;
+    width: calc(100% - var(--pf-global--spacer--lg));
+  }
+}
+
+.co-overview-card__actions {
+  align-self: center !important;
```

Comment:
Let's open an issue or bug to track when we can remove this.

Reply:
Agreed, thanks Rasto!